decidim-meetings/app/models/decidim/meetings/meeting.rb

```diff
@@ -45,11 +45,46 @@ class Meeting < Meetings::ApplicationRecord
       scope :upcoming, -> { where(arel_table[:end_time].gteq(Time.current)) }
 
       scope :visible_meeting_for, lambda { |user|
-                                    joins("LEFT JOIN decidim_meetings_registrations ON
-                                    decidim_meetings_registrations.decidim_meeting_id = #{table_name}.id")
-                                      .where("(private_meeting = ? and decidim_meetings_registrations.decidim_user_id = ?)
-                                    or private_meeting = ? or (private_meeting = ? and transparent = ?)", true, user, false, true, true).distinct
-                                  }
+        (all.distinct if user&.admin?) ||
+          if user.present?
+            user_role_queries = %w(conference assembly participatory_process).map do |participatory_space_name|
```

Comment:
be aware of the conference, it is not a mandatory module and this will break the installation if not included

Reply:
Hi @microstudi , thanks for the comment. Below the line you have made the reference, there is a begin rescue block to control all modules that are optional. In case of Conference doesn't exists then the rescue code will not implement the part of the query for this specific module.
Have you notice some execution errors in this part? If it is, can you explain how to reproduce it?
Thanks

Reply:
I think is because rescues NameError and the exception is another type

Reply:
Ok, I changed that using your comment on #6934 . I think it is a better solution than mine.
Thanks @microstudi